backend/Makefile

```diff
@@ -40,18 +40,18 @@ $(BUILD_TARGETS):
 
 
 .PHONY: test
-test: $(TEST_TARGETS) pkg-test test-integration
+test: test-unit test-integration
 
 .PHONY: $(TEST_TARGETS)
 $(TEST_TARGETS):
 	@$(MAKE) -C services/$(subst -test,,$@) test
 
 .PHONY: pkg-test
 pkg-test:
-	@$(MAKE) -C pkg/
+	@$(MAKE) -C pkg/ test
 
 .PHONY: test-unit
-test-unit: $(TEST_UNIT_TARGETS)
+test-unit: pkg-test $(TEST_UNIT_TARGETS)
```

Comment:
wiat, test-unit reqs pkg-test, pkg-test calls text and test needs test-unit. luckly from the pkg directory. good that I am not lost